src/main/webapp/groups.js

```diff
@@ -0,0 +1,16 @@
+goog.require('goog.dom');
+goog.require('goog.style');
+
+let newGroupContainer;
+
+function init() {
+    newGroupContainer = goog.dom.getElement("new-group-container");
+}
+
+function showForm() {
+    goog.style.setStyle(newGroupContainer, "display", "flex");
+}
+
+function hideForm() {
+    goog.style.setStyle(newGroupContainer, "display", "none");
```

Comment:
Sorry, I forgot to mention this earlier. You should instead have a CSS class that goes on the element that can be toggled to make the element visible/not visible.